Dockerfile

```diff
@@ -18,17 +19,24 @@ RUN /env/bin/pip install \
   rm -rf /tmp/dea_ml
 
 # Below is the actual image that does the running
-FROM opendatacube/geobase:runner
+FROM opendatacube/geobase-runner:${V_BASE}
 ARG py_env_path=/env
 
+# Environment can be whatever is supported by setup.py
+# so, either deployment, test
+ARG ENVIRONMENT=deployment
+RUN echo "Environment is: $ENVIRONMENT"
+
 ENV DEBIAN_FRONTEND=noninteractive \
     PATH="${py_env_path}/bin:${PATH}" \
     LC_ALL=C.UTF-8 \
     LANG=C.UTF-8
 
+RUN apt-get update \
+    && apt-get install software-properties-common -y \
+    && apt-get upgrade -y
```

Comment:
Not a good idea, I don't think.

Reply:
otherwise the libbooks and libcgal  can not be found